app/models/match.rb

```diff
@@ -41,6 +41,23 @@ def self.follow_up_status(follow_up_status)
     result
   end
 
+  def self.create_match_for_contribution!(contribution, current_user)
```

Comment:
Could we add the following comment above this method:
`# FIXME: extract into an interactor`.

I think this kind of use-case specific logic fits nicely into the interactor pattern which, among other benefits, helps to keep models from getting bloated. Unfortunately we don't have an example in the codebase i can point at currently (though we have pulling in the `active_interaction` gem), so probably makes sense to refactor this later.

Reply:
That is really good to know. Do you mind if I try to implement `active_interaction`? I wasn't sure if you wanted to set up any conventions for `interactors`.

Reply:
Oh yes, please go ahead if you have the time and are motivated! 💯

I don't have any strong conventions in mind, the [gem](https://github.com/AaronLasseigne/active_interaction) is pretty good.

In a separate WIP branch, i've placed them in [`app/interactions`](https://github.com/rubyforgood/mutual-aid/tree/audit-email-code/app/interactions). The example there uses the `run!` form which assumes the interactor isn't trying to return any meaningful data back to the controller; there's the regular `run` form for when a result object is useful).